docs/errors.html

```diff
@@ -272,12 +271,12 @@ <h3>Description</h3>
 By the way, why would you use an empty link?</p>
 <h3>References</h3>
 <ul>
-<li><a href="https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-6-5">https://references.modernisation.gouv.fr/rgaa/criteres.html#crit-6-5</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#text-equiv-all">https://www.w3.org/TR/WCAG20/#text-equiv-all</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs">https://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs</a></li>
-<li><a href="https://www.w3.org/TR/WCAG20/#navigation-mechanisms-link">https://www.w3.org/TR/WCAG20/#navigation-mechanisms-link</a></li>
-<li><a href="https://www.w3.org/TR/WCAG-TECHS/H30.html">https://www.w3.org/TR/WCAG-TECHS/H30.html</a></li>
-<li><a href="https://www.w3.org/TR/WCAG-TECHS/G91.html">https://www.w3.org/TR/WCAG-TECHS/G91.html</a></li>
+<li><a href="https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#6.5">https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#6.5</a></li>
```

Comment:
I'd need to test each links because this anchor doesn't existe, should be 6.2.

I think Linkinator cannot handle anchors :/

Reply:
Oops, found 3 others that can't exist